cmd/stack.go

```diff
@@ -355,12 +357,15 @@ func printStatus(cmd *cobra.Command, servicesStatus []stack.ServiceStatus) {
 		cmd.Printf(" - No service running\n")
 		return
 	}
-	t := table.NewWriter()
-	t.AppendHeader(table.Row{"Service", "Version", "Status"})
-
+	config := defaultColorizedConfig()
+	config.Settings.Separators.BetweenRows = tw.Off
+	table := tablewriter.NewTable(cmd.OutOrStderr(),
```

Comment:
I am using `cmd.OutOrStderr()` here because it is what `cmd.Println` does. I found out by seeing that `scripts/test-stack-command.sh` uses `2>` to redirect its output for comparison.

We should also work on consistency of outputs, I would say that all these tables should go to stdout.

Not to change in this PR in any case.